intro-to-git/collaborating.md

```diff
@@ -0,0 +1,171 @@
+# Collaborating
+
+## Learning Goals
+
+- Explain how git is version control system used to collaborate on code
+- Compare the difference between code on a local machine versus code on a remote repository
+- Explain how to get changes from a remote repository using git pull (or git fetch and git merge)
+- Explain how to push changes to a remote repository using git push
+- Recall that a local machine must merge in changes from the remote before pushing
+
+## Introduction
+
+Developers working on projects tracked by Git build their projects with commits. We can imagine a single developer, Nakita, working on her project named Hike Planner. By the end of the week, she's created twenty commits, and her Git history tells her the story of the code.
+
+How would Nakita's workflow change if the Hike Planner team grows? Multiple developers can work on a project, and Git has even more tools to facilitate collaborating. The key to collaborating with Git is working to converge on a single, unified Git history together.
+
+
+## Vocabulary and Synonyms
+
+| Vocab | Definition | Synonyms | How to Use in a Sentence
+| --- | --- | --- | ---
+Remote repository | asdf | Remote, remote repo, `origin` | asdf
```

Comment:
**Definition**: "repositories other than the one on your local disk which you can push your changes into (so that other people can see them) or pull from (so that you can get others changes)"
**How to Use in a Sentence**: Once I finished committing my changes, I pushed them to the **remote repository.**